src/interactive/dropdown.test.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import Section from '../layout/Section';
+import Chunk from '../layout/Chunk';
+import Button from '../forms/Button';
+
+import Dropdown from './Dropdown';
+
+const dropdownContent = (
+	<Section noSeparator>
```

Comment:
Nitpick: use `<Section className='border--none'>` instead - it gives you the correct spacing at the bottom.